examples/grpc-e2e-tests/tests/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/GoogleContainerTools/skaffold/examples/grpc-e2e-tests/tests
+
+go 1.18
```

Comment:
Please move to v1.19 due to current CVE in go.18

Reply:
Thanks , moved to 1.19.